ntbs-service/wwwroot/source/Components/CascadingDropdowns.ts

```diff
@@ -0,0 +1,31 @@
+import Vue from 'vue';
+const axios = require('axios');
+
+const CascadingDropwdowns = Vue.extend({
+    methods: {
+      fetchHospitalList: function (event: Event) {
+          console.log("triggered")

```

Comment:
Let's cut this (or move to "debug" and make more explicit :) )

Reply:
ah, forgot this one here)